doc/classes/AnimationNodeAnimation.xml

```diff
@@ -17,6 +17,9 @@
 		</member>
 		<member name="loop_mode" type="int" setter="set_loop_mode" getter="get_loop_mode" enum="Animation.LoopMode">
 			If [member use_custom_timeline] is [code]true[/code], override the loop settings of the original [Animation] resource with the value.
+			[b]Note:[/b] The [member loop_mode] does not duplicate the [Animation] with the overriding options, but rather performs a loop on the result played back on the [AnimationTree] like a post process.
+			For example, if you actually set a loop in the [Animation] or not, the behavior of setting by [method Animation.track_set_interpolation_loop_wrap] and processing the ends key or not will be different.
```

Comment:
That sentence is unclear, I've tried to improve the language a bit but I'm not sure to understand it.